tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php

```diff
@@ -40,22 +43,21 @@ function _before(AcceptanceTester $I) {
   public function _after(AcceptanceTester $I) {
     $I->logInWithRole('administrator');
     $I->amOnPage('/admin/config/system/local-footer');
-    $I->uncheckOption('#edit-su-local-foot-use-loc-value');
-    $I->uncheckOption('#edit-su-local-foot-use-logo-value');
-    $I->selectOption("#edit-su-local-foot-loc-op", "a");
+    $I->uncheckOption('Use Default Lockup');
+    $I->uncheckOption('Use the logo supplied by the theme');
+    $I->selectOption('Lockup Options', "a");
     // In case there was an image already.
-    try {
+    if ($I->grabMultiple('input[value="Remove"]')) {
```

Comment:
That is much tidier. Thanks.

Reply:
FWIW the try catch didn't work in the D9 testing. It wouldn't ever `catch`. It just failed.

Reply:
Most interesting.